src/app/docs/checkbox/checkbox-docs.component.html

```diff
@@ -0,0 +1,14 @@
+<sky-docs-demo-page
+  moduleName="SkyCheckboxModule"
+  moduleSourceCodePath="src/app/public/modules/checkbox/"
+  pageTitle="Checkbox"
+>
+  <sky-docs-demo-page-summary>
+    The checkbox component is a SKY UX-themed replacement for an HTML <sky-code>input</sky-code> element with <sky-code>type="checkbox"</sky-code>. When users select a checkbox, its value is driven through an <sky-code>ngModel</sky-code> attribute that you specify on the <sky-code>sky-checkbox</sky-code> element.
```

Comment:
This summary is longer than what Adam and Todd were shooting for in the redesign and includes the type of implementation details that they've tried to put in the Development tab. Given that we're just moving this along, maybe that doesn't matter for now and we should just leave it as is and circle back later. But FWIW, I would change this summary to: `Checkboxes enable users to select multiple items from a short list of items.` (This is taken from the guidelines at https://developer.blackbaud.com/skyux/design/guidelines/form-design#selection-inputs).

As for the implementation details that are currently in the summary, I'd move them to the `SkyCheckboxComponent` description. I'd delete the start of the first sentence and make that description: `Replaces the HTML input element with type="checkbox". When users select a checkbox, its value is driven through an ngModel attribute that you specify on the sky-checkbox element.`

Reply:
Done